### PR TITLE
created login page with bulma css

### PR DESCRIPTION
**Pull request by sourcefactory-at**
As described in #271 I added bulma css to login page and moved button labels to translation files.

Unauthenticated:
image

Authenticated:
image

### CODE REVIEW

**Comment by eerison** [comment]
Hi @sourcefactory-at yes it's really good for now :)

I'll try to review this in the next days :)

File: templates/bundles/FOSUserBundle/Security/login_content.html.twig [hunk 1]
```diff
@@ -1,59 +1,108 @@
...
-                    {% if error %}
-                        <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
-                    {% endif %}
-
-                    <button type="submit" id="_submit" name="_submit" class="btn btn-lg btn-primary btn-block">{{ 'security.login.submit'|trans }}</button>
-                    <div>
-                        <a href="{{path('fos_user_registration_register')}}">{{'layout.register'| trans}}</a> or <a href="{{path('fos_user_resetting_request')}}">{{'resetting.email.subject' | trans}}</a>
+{{ encore_entry_link_tags('web-site') }}
+
+<section class="section">
```
Comment (eerison) [inline templates/bundles/FOSUserBundle/Security/login_content.html.twig:5]: Screenshot from 2020-10-06 11-03-54


Hey @sourcefactory-at could you check when you put a wrong password, could you put this `invalid credentials` red

and I think this `register` and reset password should be side to side

**Comment by sourcefactory-at** [comment]
Hi @eerison,
the error message is now red and I moved the register and reset buttons to the right side. It depends on the screen size when they float below each other but I think now it looks ok.

image

image

**Comment by eerison** [comment]
Perfect, thank you @sourcefactory-at :)

**Comment by eerison** [comment]
@sourcefactory-at I don't know if you have time, But I think you're really able to do this issue #272 :smile:

**Pull request merged**